lib/instrumentation/mongodb/v4-mongo.js

```diff
@@ -45,7 +45,7 @@ function cmdStartedHandler(shim, evnt) {
       if (hosts && hosts.length && hosts[0].socketPath) {
         port = hosts[0].socketPath
       }
-    } else if (['127.0.0.1', '::1'].includes(host)) {
+    } else if (['127.0.0.1', '[::1]'].includes(host)) {
```

Comment:
as i mentioned in the PR details. changing this made tests pass but no previous versions fail so I'm a little confused as #1288 was a fix for this.  I'm wondering if we only had test coverage for one part of this which was supporting ipv6 resolution not passing in of ipv6 addresses but I'm not really sure

Reply:
ok turns out it's some random patch versions that change this behavior.  I'm going to restore